### PR TITLE
Updates documentation for Venafi's SSH feature

### PR DESCRIPTION
**Pull request by luispresuelVenafi**

### CODE REVIEW

**Review by jdw2465VEN — APPROVED** [review]
The changes look fine to me. Nice work, Luis.

**Review by jdw2465VEN — APPROVED** [review]

**Review by jdw2465VEN — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -293,6 +293,59 @@ using the
    named after the zone that was specified.  The 
    *certificate:manage* scope is require to import policy from TPP.
 
+## SSH Certificate Management
+
+1. Declare the Venafi Provider and specify the connection and authentication settings as described in the previous sections.
```
Comment (jdw2465VEN) [inline README.md]: One comment...don't capitalize "provider." should be "Declare the Venafi provider and specify . . . ."

**Review by rvelaVenafi — APPROVED** [review]
Looks good to me

File: README.md [hunk 2]
```diff
@@ -293,6 +293,59 @@ using the
...
+
+   >:pushpin: **NOTE**: For Trust Protection Platform, the access_token assigned to the venafi provider must have the ssh:manage scope in order to create SSH certificates.
```
Comment (jdw2465VEN) [inline README.md:300]: Minor, but capitalize "Venafi" in your NOTE

**Pull request merged**